tests/dGameTests/dComponentsTests/DestroyableComponentTests.cpp

```diff
@@ -0,0 +1,300 @@
+#include "GameDependencies.h"
+#include <gtest/gtest.h>
+
+#include "BitStream.h"
+#include "DestroyableComponent.h"
+#include "Entity.h"
+
+class DestroyableTest : public GameDependenciesTest {
+	protected:
+		Entity* baseEntity;
+		DestroyableComponent* destroyableComponent;
+		CBITSTREAM
+		uint32_t flags = 0;
+		void SetUp() override {
+			SetUpDependencies();
+			baseEntity = new Entity(15, GameDependenciesTest::info);
+			destroyableComponent = new DestroyableComponent(baseEntity);
+			baseEntity->AddComponent(COMPONENT_TYPE_DESTROYABLE, destroyableComponent);
+			// Initialize some values to be not default
+			destroyableComponent->SetMaxHealth(12345.0f);
```

Comment:
any specific reasons for these values? (this is just curiosity)

Reply:
Just arbitrary values.  No meaning.